server/web/api/register.py

```diff
@@ -69,58 +97,57 @@ def login():
 @api.get("/templates")
 @admin_only
 def get_templates():
-    return [
-        {"id": 10023, "name": "Busunfall", "players": 5},
-        {"id": 900323, "name": "Explosion im Wohnviertel", "players": 10},
-    ]
+    return [{"id": scenario.id, "name": scenario.name, "executions": [execution.id for execution in scenario.executions]}
+            for scenario in models.Scenario.query]
 
 
 @api.post("/scenario/start")
-@admin_only
+# @admin_only
```

Comment:
Commented code is bad 👻 (or should be removed before merging)

Reply:
I forgot to reenable that after testing. Should be fine now.